package.json

```diff
@@ -109,24 +109,35 @@
     "@keystonehq/bc-ur-registry-eth": "^0.12.1",
     "@keystonehq/metamask-airgapped-keyring": "^0.6.1",
     "@material-ui/core": "^4.11.0",
+    "@metamask/address-book-controller": "1.0.0",
```

Comment:
Is it intentional for the controllers packages to be using static versions? Do we not want to allow patches or minor releases? I.e. `^1.0.0`?

Reply:
Oops, this was a mistake. I will fix this.

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/16547/commits/ec2be79814db112e06a99a05ac25b8971ac7e97e